hooks/services.py

```diff
@@ -2,11 +2,21 @@
 
 from charmhelpers.core.services.base import ServiceManager
 from charmhelpers.core.services import helpers
-from charmhelpers.contrib.benchmark import Benchmark
 
 import actions
 from charmhelpers.core import hookenv
 
+try:
```

Comment:
Since you're installing this here in the charm there's no need to do it in the action :dancers: